bark/models/behavior/safety_behavior/safety_behavior.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020 fortiss GmbH
+//
+// Authors: Julian Bernhard, Klemens Esterle, Patrick Hart and
+// Tobias Kessler
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#include <tuple>
+#include <memory>
+
+#include "bark/models/behavior/safety_behavior/safety_behavior.hpp"
+#include "bark/models/behavior/idm/idm_lane_tracking.hpp"
+#include "bark/world/observed_world.hpp"
+
+namespace bark {
+namespace models {
+namespace behavior {
+
+Trajectory BehaviorSafety::Plan(
```

Comment:
same name as file names

Reply:
The RSS evaluator implementation needs some rework.. I do not know what agent_id there is and also cannot handle observed world @klemense1

Reply:
I just implemented and pushed the `Evaluate(ObservedWorld observed_worrld)` interface